src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/FromFhirToEdifactMapper.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Segment;
+import uk.nhs.digital.nhsconnect.nhais.parse.FhirParser;
+
+public interface FromFhirToEdifactMapper<T extends Segment> {
+    T map(Parameters parameters);
+
+    default Patient getPatient(Parameters parameters) {
```

Comment:
I'll remove this method as I created `ParametersExtension` that provides `extractPatient()` method (plus some additional convenience methods), but that can be done after this PR is merged